### PR TITLE
Move widgetrc to javascript and remove preventBrowserFromSavingOktaPassword option

### PR DESCRIPTION
**Pull request by haishengwu-okta**
## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Issue:

- https://oktainc.atlassian.net/browse/OKTA-329146

### CODE REVIEW

File: src/models/Settings.js [hunk 1]
```diff
@@ -66,7 +66,6 @@ export default Model.extend({
     'features.webauthn': ['boolean', true, false],
     'features.selfServiceUnlock': ['boolean', true, false],
     'features.multiOptionalFactorEnroll': ['boolean', true, false],
-    'features.preventBrowserFromSavingOktaPassword': ['boolean', true, true],
```
Comment (aarongranick-okta) [inline src/models/Settings.js:69]: was this change intentional? does not seem to belong to this PR

Comment (haishengwu-okta) [reply src/models/Settings.js:69]: yeah, I shall mentioned in the description. I meant for 2 commits to fix those 2 problems.

**Review by aarongranick-okta — APPROVED** [review]
